backend/config/default.cjs

```diff
@@ -218,4 +218,18 @@ module.exports = {
   experimental: {
     v2: false,
   },
+
+  connectors: {
+    user: {
+      kind: 'silly',
+    },
+
+    authorisation: {
```

Comment:
Could this be combined with the config that relates to the chosen auth provider. Using oauth as an example, to use oauth you have to both add the oauth section of config and also have to set the oath connector in a different place in the config. We have oauth.enabled, if that's set to true then I shouldn't also need to set the connectors.authorisation to be oauth.